python/pyspark/ml/util.py

```diff
@@ -760,3 +762,127 @@ def _get_active_session(is_remote: bool) -> SparkSession:
     if spark is None:
         raise RuntimeError("An active SparkSession is required for the distributor.")
     return spark
+
+
+class FunctionPickler:
+    """ 
+        This class provides a way to pickle a function and its arguments.
+        It also provides a way to create a pytorch script that can run a
+        function with arguments if they have pickled to a file.
+    """
```

Comment:
None of these functions are specific to training I would say. Let's reword the comments to not mention training

Reply:
Done.

Reply:
I still see mentions of training around the file